extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesConfig.java

```diff
@@ -261,15 +261,6 @@ public enum DeploymentResourceKind {
     @ConfigItem
     ResourcesConfig resources;
 
-    /**
-     * If true, a Kubernetes Ingress will be created
-     *
-     * @deprecated Use the {@code quarkus.kubernetes.ingress.expose} instead
-     */
-    @ConfigItem
-    @Deprecated
-    boolean expose;
```

Comment:
Small advice, when adding `@Deprecated`, make sure you add a `since` with the version in which they are deprecated.